packages/flutter/lib/src/rendering/list_wheel_viewport.dart

```diff
@@ -828,22 +828,51 @@ class RenderListWheelViewport
 
   /// Paints all children visible in the current viewport.
   void _paintVisibleChildren(PaintingContext context, Offset offset) {
+    if (overAndUnderCenterOpacity >= 1) {
```

Comment:
should we also go down this code path if useMagnifier is false? Seems like in that case the other code path would paint some children twice since _paintTransformedChild ignores the value of center in that case?

Reply:
see line 907

Reply:
No, the magnifier cannot be disabled if there is opacity.

Reply:
Is that enforced anywhere in the API that `useMagnifier` must be true if `overAndUnderCenterOpacity` is less than 1? Or is it if there is opacity the value of `useMagnifier` is just (silently) ignored?

This is confusing, but I guess not a problem of this PR...

Reply:
the value of use magnifier is just ignored